Machine Learning/Regression Loss Functions/regression-loss-functions.ipynb

```diff
@@ -0,0 +1,415 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = symbols('a') #actual value\n",
+    "p = symbols('p') #predicted value\n",
+    "mse = lambda a,p: (a-p)**2\n",
+    "mse_plot = plot(mse(0, p),(p, -3, 3), show=False, legend=True, line_color=\"red\")\n",
+    "mse_plot[0].label='MSE'\n",
+    "mse_plot.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The problem with the squared error is that it makes small errors (< 1.0) even smaller and large large errors (>1.0) disproportionately larger.\n",
+    "\n",
+    "This means that if you have a few outliers in your data, i.e. due to a large measurement error, the model will put useless effort to fit these data points and other points will fall by the wayside. \n",
```

Comment:
I think I'd capture that this will often degrade accuracy for the majority inliers:
```suggestion
    "This means that if you have a few outliers in your data, i.e. due to a large measurement error, the model will put useless effort to fit these data points often degrading the accuracy of the majority of predictions as a result. \n",
```

Reply:
fixed